test/integration/windows/serverspec/spec_helper.rb

```diff
@@ -0,0 +1,5 @@
+require 'serverspec'
+require 'pathname'
+
+set :backend, :cmd
+set :os, family: 'windows'
```

Comment:
Move to `test/integration/helpers/serverspec/spec_helper` and the spec_helper will automatically be shared by all suites. Example here:
https://github.com/chef-cookbooks/languages/blob/master/test/integration/helpers/serverspec/spec_helper.rb